regulations/generator/html_builder.py

```diff
@@ -207,6 +208,8 @@ def human_label(node):
 
     def process_node(self, node, indexes=None):
         """Overrides with custom, additional processing"""
+        if indexes is None:
+            indexes = self.index_prefix
```

Comment:
Minor but rather than putting this logic in `process_node`, would it make sense to just add is as the final param in `generate_html`?